packages/grafana-data/src/dataframe/MutableDataFrame.ts

```diff
@@ -113,6 +113,7 @@ export class MutableDataFrame<T = any> implements DataFrame, MutableVector<T> {
       config: f.config || {},
       values: this.creator(buffer),
       labels: f.labels,
+      display: (f as Field).display,
```

Comment:
Do we need as Field typing?
```suggestion
      display: f.display,
```

Reply:
can be either `Field` or `FieldDTO`

Reply:
ok, I see